src/stirling/source_connectors/socket_tracer/socket_trace_connector.h

```diff
@@ -65,6 +65,7 @@ DECLARE_int32(stirling_enable_nats_tracing);
 DECLARE_int32(stirling_enable_kafka_tracing);
 DECLARE_int32(stirling_enable_mux_tracing);
 DECLARE_int32(stirling_enable_amqp_tracing);
+DECLARE_int32(stirling_enable_mqtt_tracing);
```

Comment:
Changes to this file should also come in a later PR (when the MQTT trace bpf test is added).